index.html

```diff
@@ -90,32 +90,38 @@ <h2 class="text-center title-medium">Features</h2>
           <ul class="row">
             <li class="col col-4">
               <img class="image-features" src="/images/money.jpg" alt="Money in hands">
-            <div class="element-in-image">
-              <h3 class="title-small">Supports All Currencies and Cards</h3>
-              <p>We support all popular currencies and is fully customizable to add</p>
-              <a class="features-link-text" href=""><p class="text-features">Read More</p></a>
-            </div>
+              <div class="element-in-image">
+                <h3 class="title-small">Supports All Currencies and Cards</h3>
+                <p>We support all popular currencies and is fully customizable to add</p>
+                <a class="features-link-text" href="">
+                  <span>Read More</span>

```

Comment:
Why do you need span? We can put text inside link without any additional wrappers.